### PR TITLE
Update Terms of Service page

### PR DESCRIPTION
**Pull request by TurnerRiley**
# NOTE: MERGE AFTER DTP SO TRAVIS CAN REVIEW ON STAGING

Updates the Terms of Service page.

localhost code org_3000_tos

## Links
Doc of changes: [here](https://docs.google.com/document/d/1-L1laZdinbmiBE80ki5vv9I891NcRmUUXHdiz2dXzaE/edit)
Jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-1688)

## Testing story
Local testing.

## Deployment strategy
I'll let Travis know once it's merged so he can proof it on staging before it goes live!

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/tos.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ These Terms are subject to change. You are agreeing to be bound by such changes
...
+We take steps to minimize the collection of personal data from Users under the age of 13 as explained in the Code.org [Privacy Policy](/privacy). Users under the age of 13 will be able to use our App Lab, Game Lab, and Web Lab tools in the classroom with their teacher’s approval. Users under the age of 13 will not be permitted to use these tools for outside the classroom use. Users under 13 can try most tutorials and courses without creating an account, but their learning progress won’t be saved. Although we encourage parental oversight for all Users under 13, depending on jurisdiction, users under 13 may be required to get the approval of a parent or legal guardian before creating an account on Code.org. Code.org allows certain representatives of school districts and schools, such as teachers, administrators, counselors, and other educators (an “**Educator**”) to connect with their students through the Services in order to provide tutorial, educational and similar services. If you are an Educator, you may register accounts on the Services for one or more of your students.
```
Comment (hannahbergam) [inline pegasus/sites.v3/code.org/public/tos.md]: Slightly surprised the first 'Users' is capitalized but I get if that's a legal thing!

Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/tos.md]: Can we update the 'users under 13' to be capitalized to match?

Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/tos.md]: And then one tiny nit- can we change 'for outside the classroom use' to 'for use outside the classroom'?

File: pegasus/sites.v3/code.org/public/tos.md [hunk 2]
```diff
@@ -72,17 +72,17 @@ You hereby grant to Code.org a limited, non-exclusive, sublicensable (as necessa
...
+In addition to the rights, licenses and privileges referred to above, you agree that Code.org may use and refer to your User Content, first name and screenname that are posted publicly on the Code.org Website and not in private areas of our Service (“Public Content”) (including screenshots) in marketing materials, press releases, financial reports, presentations, website materials, customer lists and other media now known or hereafter discovered in connection with the marketing, advertising and promotion of the Service, and any products, goods, features, capabilities and/or services associated with the Service. This right will continue until your Public Content is deleted; provided however, that any Public Content previously printed in brochures, articles or other marketing materials that Code.org had prepared before any request for deletion was received will continue to be used.
```
Comment (hannahbergam) [inline pegasus/sites.v3/code.org/public/tos.md]: Now I'm not sure what a 'screenname' is but I assume it's also a legal term

**Review by hannahbergam — APPROVED** [review]
Only tiny nits! Admittedly, my eyes went blurry by the end but I did a close read of most of it!

File: pegasus/sites.v3/code.org/public/tos.md (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/tos.md]: Maybe it's Username? If yes, I wonder if this should be changed (probs a Travis call).

**Review by kelbyhawn — APPROVED** [review]
I agree with Hannah's suggestions, but looks good to go! 🎉

File: pegasus/sites.v3/code.org/public/tos.md (hunk 1 above)
Comment (TurnerRiley) [reply pegasus/sites.v3/code.org/public/tos.md]: I believe "Users" being capitalized is a legal thing with the way some of the other words like "Child" are being used in the other docs. Yes good call I'll make that consistent and update the phrasing!

File: pegasus/sites.v3/code.org/public/tos.md (hunk 2 above)
Comment (TurnerRiley) [reply pegasus/sites.v3/code.org/public/tos.md]: I'll ask, good call!

**Pull request merged**